pkg/rbac/parser.go

```diff
@@ -44,8 +48,62 @@ type Rule struct {
 	URLs      []string `marker:"urls,optional"`
 }
 
+// target generates a string representing the resources and/or non-resources a Rule applies.
```

Comment:
this godoc should be a bit more specific on what's used and why.

Reply:
Removed this method, and use `ruleKey` as the map key.